cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslHelp.groovy

```diff
@@ -0,0 +1,33 @@
+package org.jenkinsci.plugins.workflow.cps.Snippetizer
+
+
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+st.contentType(value: "text/html;charset=UTF-8")
+
+l.layout(title:_("Jenkins Workflow Reference"),permission:app.READ) {
```

Comment:
Without `Jenkins.READ` you could not get to this view via Stapler to begin with, so the `permission` is redundant.